config-example/config.txt

```diff
@@ -28,6 +28,9 @@ JOBS_HAVE_MINIMAL_ACCESS
 ## Unhash this to use recursive explosions, keep it hashed to use circle explosions. Recursive explosions react to walls, airlocks and blast doors, making them look a lot cooler than the boring old circular explosions. They require more CPU and are (as of january 2013) experimental
 #USE_RECURSIVE_EXPLOSIONS
 
+## Automatically give localhost clients +HOST access
```

Comment:
Please add a "FUCKING DISABLE THIS ON PRODUCTION SERVERS OR YOU WILL GET REKT" comment